bundles/org.openhab.binding.speedtest/src/main/java/org/openhab/binding/speedtest/internal/dto/ResultContainer.java

```diff
@@ -0,0 +1,402 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.speedtest.internal.dto;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link ResultContainer} class defines a container for Speedtest results.
+ *
+ * @author Brian Homeyer - Initial contribution
+ */
+public class ResultContainer {
+
+    @SerializedName("type")
+    @Expose
+    private String type;
+    @SerializedName("timestamp")
+    @Expose
+    private String timestamp;
+    @SerializedName("ping")
+    @Expose
+    private Ping ping;
+    @SerializedName("download")
+    @Expose
+    private Download download;
+    @SerializedName("upload")
+    @Expose
+    private Upload upload;
+    @SerializedName("packetLoss")
+    @Expose
+    private Double packetLoss;
+    @SerializedName("isp")
+    @Expose
+    private String isp;
+    @SerializedName("interface")
+    @Expose
+    private Interface _interface;
```

Comment:
Minor checkstyle you want to adjust.

Reply:
fixed. `interface` cannot be used - conflict. using `networkInterface` instead.